.rubocop_todo.yml

```diff
@@ -117,11 +117,6 @@ Metrics/ClassLength:
 Metrics/CyclomaticComplexity:
   Max: 24
 
-# Offense count: 60
-# Configuration parameters: AllowURI, URISchemes.
-Metrics/LineLength:
-  Max: 6121
-
```

Comment:
Yes, it complains about long lines. And indeed, in a future commit, someone should fix that. I'd really rather not muddle getting `master` and `develop` back in sync with additional changes.

I'd also be concerned more if `dor-services` had a clean bill of health from rubocop, but there are already 200 violations (and the vast majority are not line length complaints).